src/transformers/testing_utils.py

```diff
@@ -654,21 +655,42 @@ def get_env(self):
         env["PYTHONPATH"] = ":".join(paths)
         return env
 
-    def get_auto_remove_tmp_dir(self, tmp_dir=None, after=True, before=False):
+    def get_auto_remove_tmp_dir(self, tmp_dir=None, before=None, after=None):
         """
         Args:
             tmp_dir (:obj:`string`, `optional`):
-                use this path, if None a unique path will be assigned
-            before (:obj:`bool`, `optional`, defaults to :obj:`False`):
-                if `True` and tmp dir already exists make sure to empty it right away
-            after (:obj:`bool`, `optional`, defaults to :obj:`True`):
-                delete the tmp dir at the end of the test
+                if :obj:`None`:
```

Comment:
@sgugger, any purpose for why does  `utils/style_doc.py` change the docstring is this strange way - I meant no new lines there - why do we need them?

Reply:
Also should it perhaps say which file(s) should be restyled when it fails?
```
Traceback (most recent call last):
  File "utils/style_doc.py", line 465, in <module>
    main(*args.files, max_len=args.max_len, check_only=args.check_only)
  File "utils/style_doc.py", line 453, in main
    raise ValueError(f"{len(changed)} files should be restyled!")
ValueError: 1 files should be restyled!
```

Reply:
Lists should always being with a new line before otherwise sphinx (sometimes) throws warnings. That's why the script wants to add them.

As for the warnings, I copied what black does (and it does not say which file should be restyled) :-)